cloud/aws.json

```diff
@@ -6,5 +6,11 @@
     "RAM": "0.004237",
     "spotRAM": "0.000892",
     "spotLabel": "kops.k8s.io/instancegroup",
-    "spotLabelValue": "spotinstance-nodes"
+    "spotLabelValue": "spotinstance-nodes",
+    "awsServiceKeyName": "AKIAXW6UVLRRTBCUKQFP",
```

Comment:
I think some documentation/comments would be useful here. Let me know what you feel should live in code. 

I propose we do two things: 1) comment that these specific fields are only needed if you want a spot feed and 2) help people understand exactly what these are and how to set them. The second part may just be referencing AWS documentation.

Reply:
Let's just do this in a README somewhere.